src/Compilers/CSharp/Test/Semantic/Semantics/UsingStatementTests.cs

```diff
@@ -87,10 +85,271 @@ static void Main()
 ";
 
             CreateCompilation(source).VerifyDiagnostics(
-                // (6,16): error CS1674: 'method group': type used in a using statement must be implicitly convertible to 'System.IDisposable'
+                // (6,16): error CS1674: 'method group': type used in a using statement must have a public void-returning Dispose() instance method.
                 Diagnostic(ErrorCode.ERR_NoConvToIDisp, "Main").WithArguments("method group"));
         }
 
+        [Fact]
+        public void UsingPatternTest()
```

Comment:
It looks like there may be a duplicate test above that used to be failing but is now succeeding. You could probably remove a few identical tests.